setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup, find_packages
+
+
+setup(
+    name="openpaygo-token", 
+    packages=find_packages(),
+    version='2.2.0',
+    license='Apache 2.0',
+    author="Solaris Offgrid",
+    url='https://github.com/openpaygo/OpenPAYGO-Token',
```

Comment:
This points to the forks URL, but I think it's easier to change in a separate PR.